opendrift/readers/slf2NC.py

```diff
@@ -0,0 +1,210 @@
+###
```

Comment:
Is this file still needed?
If so, it should get a more explicit/descriptive name, and also be moved to the [scripts](https://github.com/OpenDrift/opendrift/tree/master/opendrift/scripts) folder.

Reply:
no that's an old one, I didn't mean to push it. That's a mistake